src/NovelAudioChannel.h

```diff
@@ -0,0 +1,27 @@
+//
+// Created by krjoh on 03/09/2019.
+//
+#ifndef NOVELRT_NOVELAUDIOCHANNEL_H
+#define NOVELRT_NOVELAUDIOCHANNEL_H
+
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_mixer.h>
+#include <iostream>
+#include <map>
+
+namespace NovelRT{
+class NovelAudioChannel {
+  public:
+    NovelAudioChannel(std::string input, bool isBgm, int channel);
+    void play();
+    Mix_Chunk* getChunk();
```

Comment:
Raw pointers are not that great. I know I'm using them in the layering service but that's because I hadn't quite figured out unique pointers for our use case (yet, though I think that may change very soon). Prefer `unique_ptr<T>` or `shared_ptr<T>` depending on use case.